pyccl/halos/profiles_2pt.py

```diff
@@ -111,7 +111,7 @@ def fourier_2pt(self, prof, cosmo, k, M, a,
             k (float or array_like): comoving wavenumber in Mpc^-1.
             M (float or array_like): halo mass in units of M_sun.
             a (float): scale factor.
-            prof2 (:class:`~pyccl.halos.profiles.HaloProfile`):
+            prof2 (:class:`~pyccl.halos.profiles.HaloProfile` or None):
```

Comment:
I just realised this actually has to be `HaloProfileHOD`, not just `HaloProfile`

Reply:
ok, fixed.